src/App.css

```diff
@@ -0,0 +1,25 @@
+.App-header {
+  background-color: #282c34;
+  min-height: 100vh;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  font-size: calc(10px + 2vmin);
+  color: white;
+}
+
+.App-link {
+  color: #61dafb;
+}
+.splash {
+  display: flex;
+  flex-wrap: wrap;
+  align-items: center;
+  justify-content: center;
+  height: 712px;
+}
+.bodyClass {
+  position: relative;
+  top: 6rem;
+}
```

Comment:
can you please use sass and use variables for colors?

Reply:
used scss and variables for colors.